sdk/storage/azure-storage-blobs/test/append_blob_client_test.cpp

```diff
@@ -327,19 +327,21 @@ namespace Azure { namespace Storage { namespace Test {
     Blobs::StartCopyBlobFromUriOptions copyOptions;
     copyOptions.ShouldSealDestination = false;
     auto copyResult = blobClient2.StartCopyFromUri(blobClient.GetUrl() + GetSas(), copyOptions);
-    // TODO: poller wait here
-    getPropertiesResult = blobClient2.GetProperties();
+    getPropertiesResult = copyResult->PollUntilDone(std::chrono::seconds(1));
```

Comment:
What's the SLA for this operation?

Reply:
Why do we need an SLA? SLA for what?

Reply:
You are waiting for an LRO to finish in test code, there should be a guaranteed ETA so that this won't take more than a certain time period that the whole test can tolerate.

Reply:
Is there is an SLA, the SLA should be guaranteed by server side, not client side. If the customer becomes impatient, that's when `context` comes into use.

I don't see anything that should be done by storage client code.

Reply:
Not in product code, but definitely in test code right?
What if this call took 1 hour to finish, do you think it is a good practice to wait 1 hour in this line?

Reply:
> What if this call took 1 hour to finish, do you think it is a good practice to wait 1 hour in this line?

If it did happen, I'd definitely do something. But I don't think it's gonna happen ever.

Reply:
So how long is the maximum execution time of this line of code? To put it in another way:
>What's the SLA for this operation?

Reply:
I don't know, this question should be answered by server guys. I only work on client side.